### PR TITLE
Feature: Tensorflow relational operators

### PR DESCRIPTION
**Pull request by stilistik**
Implements mappings for the TF binary relational operators 'Less', 'LessEqual', 'Greater' and 'GreaterEqual' to NNVM operators and contributes the corresponding tests.

### CODE REVIEW

**Comment by tqchen** [comment]
@liangfu @nishi-t @PariksheetPinjari909 can you help reveiw this?

**Comment by tqchen** [comment]
please rebase against the most recent master to resolve the conflict

**Comment by liangfu** [comment]
@stilistik can you add Equal and NotEqual operators as well?

File: nnvm/python/nnvm/frontend/tensorflow.py [hunk 1]
```diff
@@ -680,6 +680,34 @@ def _impl(inputs, attr, params):
         return _sym.Variable(name=name, shape=params[name].shape)
     return _impl
 
+def _less():
```
Comment (nishi-t) [inline nnvm/python/nnvm/frontend/tensorflow.py]: Could you implement helper function and use it?

for example: 
````python
def _broadcast(name):
    def _impl(inputs, attr, params):
        return AttrCvt(
            op_name=name,
            ignores=['name', 'Tidx'])([inputs[0], inputs[1]], attr)
    return _impl
````

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -877,10 +877,91 @@ def test_forward_transpose():
     _test_forward_transpose((2, 3, 4), (0, 1, 2))
     _test_forward_transpose((2, 3, 4, 5), (3, 0, 1, 2))
 
+#######################################################################
+# Less
+# ----
+def _test_forward_less(data):
```
Comment (nishi-t) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: I think these test(less, less_equal, greater, greater_equal) are very similar. Could you put these in one as with [this](https://github.com/dmlc/tvm/blob/master/nnvm/tests/python/frontend/tensorflow/test_forward.py#L333-L349)?

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 3]
```diff
@@ -877,10 +877,91 @@ def test_forward_transpose():
...
+#######################################################################
+# less equal
+# ----------
+def _test_forward_less_equal(data):
+    """ One iteration of LessEqual operation """
+
+    with tf.Graph().as_default():
+        in1 = tf.placeholder(tf.int32, shape=[3, 3], name='in1')
+        in2 = tf.placeholder(tf.int32, shape=[3, 3], name='in2')
+        gte = math_ops.less_equal(in1, in2, name='gte')
```
Comment (PariksheetPinjari909) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Use name 'le' for less equal operation. And better use same variable name

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 4]
```diff
@@ -877,10 +877,91 @@ def test_forward_transpose():
...
+        out1 = tf.cast(gte, dtype=tf.int32, name='out1')
+
+        compare_tf_with_tvm([data[0], data[1]], ['in1:0', 'in2:0'], 'out1:0')
+
+def test_forward_less_equal():
+    t1 = np.array([[1, 2, 3], [4, 5, 6], [8, 9, 10]])
+    t2 = np.array([[7, 1, 2], [4, 5, 6], [3, 10, 1]])
+    _test_forward_greater_equal([t1, t2])
```
Comment (PariksheetPinjari909) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Call _test_forward_less_equal() here

**Comment by stilistik** [comment]
> @stilistik can you add Equal and NotEqual operators as well?
Featured in my latest commit

File: nnvm/python/nnvm/frontend/tensorflow.py (hunk 1 above)
Comment (stilistik) [reply nnvm/python/nnvm/frontend/tensorflow.py]: Featured in my latest commit

**Comment by tqchen** [comment]
Thanks @nishi-t @stilistik @PariksheetPinjari909 @liangfu this is now merged

**Pull request merged**